website/pages/component/box/example.tsx

```diff
@@ -13,28 +13,29 @@ const knobsConfig: KnobsConfigInterface<{
     default: 'div',
     label: 'As',
   },
-  href: {
-    type: 'select',
-    options: ['mongodb.design', undefined],
-    default: undefined,
-    label: 'href',
-  },
   children: {
     type: 'text',
     default: 'Box component',
     label: 'Children',
   },
+  href: {
+    type: 'boolean',
+    default: false,
+    label: 'href',
+  },
 };
 
 export default function BoxLiveExample() {
   return (
     <LiveExample knobsConfig={knobsConfig}>
-      {({ as, href, children }) => (
-        // @ts-ignore
-        <Box as={as} href={href}>
-          {children}
-        </Box>
-      )}
+      {({ as, children, href }) => {
+        if (href) {
+          // @ts-expect-error
+          return <Box href={href}>{children}</Box>;
```

Comment:
we should disable the knob for `as` when `href` is `true`, otherwise its value being ignored would be unexpected behavior when using the live example.

Reply:
👍